src/components/SummaryCard/SummaryCardBatchActions/__tests__/SummaryCardBatchActions.spec.js

```diff
@@ -9,10 +9,14 @@ import React from 'react';
 import { SummaryCardBatchActions } from '../../../..';
 
 describe('SummaryCardBatchActions', () => {
-  it('renders', () => {
-    expect(
-      render(<SummaryCardBatchActions onCancel={jest.fn()} totalSelected={0} />)
-        .container.firstChild
-    ).toMatchSnapshot();
+  test('should pass through extra props via spread attribute', () => {
```

Comment:
Will we need a11y tests for these? Also, we'll most likely also want to cover `translateWithId`

Reply:
Will we need a11y tests for these?

Reply:
This is a Carbon prop though? Also, the a11y test is in `SummaryCard`